content/en/docs/features/initdata/_index.md

```diff
@@ -0,0 +1,170 @@
+
+- `url` of `[token_configs.kbs]` section in `"aa.toml"`, defining the KBS for remote attestation.
+- `url` of `[kbc]` section in `"cdh.toml"`, defining the KBS for accessing confidential resources. In this case, we use the same KBS for both attestation and resource provision.
+
+```toml
+version = "0.1.0"
+algorithm = "sha256"
+[data]
+"policy.rego" = '''
+# Copyright (c) 2023 Microsoft Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+package agent_policy
+
+default AddARPNeighborsRequest := true
+default AddSwapRequest := true
+default CloseStdinRequest := true
+default CopyFileRequest := true
+default CreateContainerRequest := true
+default CreateSandboxRequest := true
+default DestroySandboxRequest := true
+default ExecProcessRequest := true
+default GetMetricsRequest := true
+default GetOOMEventRequest := true
+default GuestDetailsRequest := true
+default ListInterfacesRequest := true
+default ListRoutesRequest := true
+default MemHotplugByProbeRequest := true
+default OnlineCPUMemRequest := true
+default PauseContainerRequest := true
+default PullImageRequest := true
+default ReadStreamRequest := true
+default RemoveContainerRequest := true
+default RemoveStaleVirtiofsShareMountsRequest := true
+default ReseedRandomDevRequest := true
+default ResumeContainerRequest := true
+default SetGuestDateTimeRequest := true
+default SetPolicyRequest := true
+default SignalProcessRequest := true
+default StartContainerRequest := true
+default StartTracingRequest := true
+default StatsContainerRequest := true
+default StopTracingRequest := true
+default TtyWinResizeRequest := true
+default UpdateContainerRequest := true
+default UpdateEphemeralMountsRequest := true
+default UpdateInterfaceRequest := true
+default UpdateRoutesRequest := true
+default WaitProcessRequest := true
+default WriteStreamRequest := true
+'''
+"aa.toml" = '''
+[token_configs]
+[token_configs.kbs]
+url = "http://1.2.3.4:8080"
+'''
+
+"cdh.toml" = '''
+[kbc]
+name = "cc_kbc"
+url = "http://1.2.3.4:8080"
+'''
+```
+
+#### Step 2: Embed the Init-Data in Pod YAML
+
+We can `gzip` compress and `base64` encode the Init-Data TOML.
+
+```bash
+initdata=$(cat initdata.toml | gzip | base64 -w0)
+```
+
+Then embed it in `pod.yaml`
+
+```bash
+cat <<EOF > pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: encrypted-image
+  annotations:
+    io.containerd.cri.runtime-handler: kata-qemu-tdx
+    io.katacontainers.config.hypervisor.kernel_params: ' agent.guest_components_procs=confidential-data-hub'
+    io.katacontainers.config.runtime.cc_init_data: ${initdata}
+spec:
+  runtimeClassName: kata-qemu-tdx
+  containers:
+    - name: test-container
+      image: some-registry.io/image
+      imagePullPolicy: Always
+      command:
+        - sleep
+        - "30"
+EOF
+```
+
+Once deployed, the pod will perform attestation with `http://1.2.3.4:8080` whenever confidential resources need to be accessed.
+
+```bash
+kubectl apply -f pod.yaml
+```
+
+
+### More-Use-Cases
```

Comment:
This can be something like "### Example-2: Specify container registry configuration"
or as @fitzthum mentioned, you move it to top level feature section

Reply:
Let me put the new version. PTAL if it is good : )